docs/src/experiment_setup_guide.md

```diff
@@ -39,13 +39,13 @@ First, create a folder for your experiment with a descriptive experiment id in t
 This is the ClimaAtmos configuration which will be used to run all simulations in the calibration. The only changes between model runs are the parameters selected for calibration.
 This file is a typical Atmos YAML file with a few restrictions. It must:
 
-- be named `atmos_config.yml`. 
+- be named `model_config.yml`.
 - include an `output_dir` to store model output.
 - include the path to the restart file. This is detailed in the next section.
 Here is an example of the arguments you will need:
 ```
 output_dir: output/experiment_name
-restart_file: experiments/experiment_name/restart_file.hdf5 
+restart_file: experiments/experiment_name/restart_file.hdf5
```

Comment:
ClimaAtmos run restarts are controlled by the `RESTART_FILE` env variable - I don't believe that these are ingested as environment variables based on config dict arguments right now (we set the ENV_VAR in the buildkite pipeline files). cc @nefrathenrici

Reply:
That is correct, but CalibrateAtmos sets the ENV variable from this config variable - see [here](https://github.com/CliMA/CalibrateAtmos.jl/blob/main/src/atmos_interface.jl#L37).

I have been meaning to set the restart file from the config directly in ClimaAtmos. I will try and find the time this week.

Reply:
Ah - this is internal to the `model_interface` below with the `ENV` variable set within the `get_config` function; no issues then.